ci_framework/roles/cifmw_cephadm/templates/ceph_rgw.yml.j2

```diff
@@ -0,0 +1,28 @@
+---
+service_type: rgw
+service_id: rgw
+service_name: rgw.rgw
+placement:
+  hosts:
+{% for host in _hosts %}
+  - {{ host }}
+{% endfor %}
+networks:
+- {{ cifmw_cephadm_rgw_network }}
+spec:
+  rgw_frontend_port: 8082
```

Comment:
This is the haproxy backend, no need for any iptables as we don't reach RGW directly, we rather use the $VIP:$PORT (port is `8080` on L24 and we should have the iptables already set for it).